source/stylist/style.py

```diff
@@ -45,7 +45,11 @@ def check(self,
         logging.getLogger(__name__).info('Style: ' + self.__class__.__name__)
         issues: List[stylist.issue.Issue] = []
         for rule in self._rules:
-            issues.extend(rule.examine(source))
+            additions = rule.examine(source)
+            issues.extend(additions)
+            result = "Failed" if len(additions) > 0 else "Passed"
```

Comment:
Could use implicit boolianness for `additions` here `result = "Failed" if additions else "Passed"`

Reply:
Done